### DISCUSSION TITLE
How to put the PHP extension "intl" ON for Joomla 5 requirement on MAC Xampp?

### DISCUSSION THREAD

**Discussion by Chacapamac** [templaza/astroid-framework#676] [Q&A]
"intl" extension is a  Joomla 5 requirement and it seem so complicated to put on in Xampp on Mac. 
Anybody have experience wit this?

**This is the BIng Chat GPT answer???????**
Are we serious here, it was so easy on MAMP.......

> If the `intl` extension still does not load
> 
> To enable the PHP extension "intl" on XAMPP for Mac, you can follow these steps:
> 
> 1. Open the Terminal application on your Mac.
> 2. Check the path of your PHP installation by typing:
>    ```
>    $ which php
>    ```
>    You should see a path like `/Applications/XAMPP/xamppfiles/bin/php`. If you see `/usr/bin/php`, you'll need to change the path to point to XAMPP's PHP.
> 3. Install `icu4c`, which is a prerequisite for the `intl` extension, using Homebrew:
>    ```
>    $ brew install icu4c
>    ```
> 4. Install the `intl` extension using PECL:
>    ```
>    $ sudo pecl update-channels
>    $ sudo pecl install intl
>    ```
>    During the installation, it will prompt you to input the `icu4c` path.
> 5. Once installed, add the following line to your `php.ini` file:
>    ```
>    extension=intl.so
>    ```
> 6. Restart Apache to apply the changes.
> 
> Make sure to locate the correct `php.ini` file used by XAMPP, which is typically found in `/Applications/XAMPP/xamppfiles/etc/`.
> 
> After restarting Apache, you can verify that the `intl` extension is enabled by running:
> ```
> $ php -m | grep intl
> ```
> This command should return 'intl', indicating that the extension is active.
> 
> Remember, if you encounter any issues during the installation, such as messages about missing releases or other errors, you may need to consult additional resources or troubleshooting guides specific to those errors. The steps provided are a general guide and may require adjustments based on your specific setup and the version of PHP you are using with XAMPP. If you're using a different version of PHP, you might need to install a version-specific `intl` package via Homebrew. For example, for PHP 7.0, you would use:
> ```
> $ brew install php70-intl
> ```
> Adjust the command according to your PHP version accordingly.

**Comment by sonvnn** [comment]
Click Open Application Folder
![Screen Shot 2024-05-08 at 09 16 09](https://github.com/templaza/astroid-framework/assets/3204103/1610fa04-d87a-416e-96c8-347a8df4ea18)

First of all stop the xampp and then kindly remove the starting semicolon ( ; ) from your xamppfiles/etc/php.ini the following code.

`;extension=php_intl.dll`

And then restart your xampp

**Comment by Chacapamac** [comment]
**Thank You Sonny., are you on Mac or PC?**

I try that solution before without success, but retrying again... Just to be sure...
![Screen Shot 2024-05-09 at 9 21 48 AM](https://github.com/templaza/astroid-framework/assets/1907276/25b919db-1fd6-4685-bc4f-fc95b178cabe)

**I stop all servers, clear all caches, and restart All servers.**

![Screen Shot 2024-05-09 at 9 28 47 AM](https://github.com/templaza/astroid-framework/assets/1907276/749832ce-cb35-48f1-8bf2-095deb2dbad8)
